.github/workflows/web-service.yml

```diff
@@ -75,10 +84,7 @@ jobs:
         run: |
           tar -xvf web-service.tar
           cd ./web-service
-          TAG="branch-$(git rev-parse --abbrev-ref HEAD)"
-          if [[ "${TAG}" == "branch-HEAD" ]]; then
-            TAG="$(git rev-parse --short HEAD)"
-          fi
+          TAG=${{ github.event.inputs.tag }}
```

Comment:
instead of requiring the tag to be provided, could we just use it when it's provided, and fallback on the branch name?

Reply:
yes we can sure.